scripts/goListFiles.sh

```diff
@@ -1,30 +0,0 @@
-#!/bin/bash
-#
-# Copyright Greg Haskins All Rights Reserved.
-#
-# SPDX-License-Identifier: Apache-2.0
-#
-
-
-find_golang_src() {
-    find $1 -name "*.go" -or -name "*.h" -or -name "*.c" -or -name "*.s"
-}
-
-list_deps() {
-    target=$1
-
-    deps=`go list -f '{{ join .Deps "\n" }}' $target`
-
-    find_golang_src $GOPATH/src/$target
-
-    for dep in $deps;
-    do
-        importpath=$GOPATH/src/$dep
-        if [ -d $importpath ];
-        then
-            find_golang_src $importpath
-        fi
-    done
-}
-
-list_deps $1 | sort | uniq
```

Comment:
it's separated out now, but this should probably be some separate pr or jira.

Reply:
It is a separate commit and has a separate jira. If it was a huge deletion would have made it into its own PR. But if you want I can separate it into its own PR. I found this in the process of making all the scripts work with go modules

Reply:
nah, no big deal. it's here and looks fine.